tests/oracle/test-oracle.cpp

```diff
@@ -1387,6 +1387,69 @@ TEST_CASE("Bulk iterators", "[oracle][bulkiters]")
     sql << "drop table t";
 }
 
+struct blob_table_creator : public table_creator_base
+{
+    blob_table_creator ( soci::session &sql ) : table_creator_base ( sql )
+    {
+        sql << "create table soci_test ("
+               "    id number(10) not null,"
+               "    img blob"
+               ")";
+    }
+};
+
+TEST_CASE ( "Oracle blob", "[oracle][blob]" )
```

Comment:
What exactly is this test case testing that is not covered in the test cases shared across backends? The reasoning for having a Oracle-specific Blob test case should probably be written down here as a comment

Reply:
I have restored the code from previous commits. We can remove it and use common test, but we need add test cases using blob before start transaction and after commit/rollback

Reply:
> we need add test cases using blob before start transaction and after commit/rollback

This would be an Oracle-specific test then as this is undefined behavior in a cross-DB setting (as specified in SOCI docs). So for this specific scenario, a backend-specific test case makes sense